includes/embeds/class-amp-youtube-embed.php

```diff
@@ -64,6 +64,12 @@ public function shortcode( $attr ) {
 	}
 
 	public function oembed( $matches, $attr, $url, $rawattr ) {
+		// Use the standard oembed handler if it's a playlist
+		$parsed_url = wp_parse_url( $url );
+		if ( isset( $parsed_url['path'] ) && false !== strpos( $parsed_url['path'], 'playlist' ) ) {
+			return wp_oembed_get( $url );
```

Comment:
That's a very good idea :)

Thoughts on how we could catch this version of the playlist URL as well: `https://www.youtube.com/embed/videoseries?list=PL9tY0BWXOZFvR0U9MnfnQSMym0fFgL5cT`?